docs/Upgrade-Guide/upgrade_to_8.md

```diff
@@ -0,0 +1,42 @@
+# Upgrade procedure to Gluster 8, from Gluster 7.x, 6.x, 5.x, 4.1.x, and 3.12.x
```

Comment:
I'm not sure if we support upgrade from 3.12. We usually support for n-3 to n. IMO we can avoid this commitment.